recipes/thrift/all/conanfile.py

```diff
@@ -64,23 +81,21 @@ def configure(self):
             del self.options.fPIC
 
     def requirements(self):
-        self.requires("boost/1.77.0")
-
-        if self.options.with_qt:
-            # FIXME: missing qt recipe
-            raise ConanInvalidConfiguration("qt is not (yet) available on cci")
-
+        self.requires("boost/1.78.0")
         if self.options.with_openssl:
-            self.requires("openssl/1.1.1l")
+            self.requires("openssl/1.1.1m")
         if self.options.with_zlib:
             self.requires("zlib/1.2.11")
         if self.options.with_libevent:
             self.requires("libevent/2.1.12")
 
-    def build_requirements(self):
-        build_os = self.settings_build.os if hasattr(self, "settings_build") else self.settings.os
+    def validate(self):
+        if self.options.with_qt:
+            # FIXME: missing qt recipe
+            raise ConanInvalidConfiguration("qt is not (yet) available on cci")
```

Comment:
but it's not missing 👁️

Reply:
yes but out of scope of this PR, I've kept previous message 😉